qemu.mk

```diff
@@ -32,6 +32,19 @@ all-clean: bios-qemu-clean busybox-clean linux-clean optee-os-clean \
 ################################################################################
 # QEMU
 ################################################################################
+QEMU_CONFIGURE_PARAMS = --target-list=arm-softmmu --cc="$(CCACHE)gcc" \
+						--extra-cflags="-Wno-error"
+
+QEMU_VIRTFS_ENABLE		?= n
```

Comment:
That would work, it's just that `mnt` and/or `media` on most systems today is the root folder for mounted partitions etc. I.e, there is almost always a sub-folder for the actual mount. But for this purpose and how we're using it we won't probably be interested in mounting other stuff.